WeatherApp/src/main/java/org/example/LocationMenu.java

```diff
@@ -0,0 +1,40 @@
+package main.java.org.example;
+
+import java.util.Scanner;
+
+public class LocationMenu {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        Location userLocation = new Location();
```

Comment:
Jak pisałem w komentarzu przy klasie, Location (czy jak to tam się będzie ostatecznie nazywało) ma być finalne, więc będziesz musiał refactorować proces tworzenia.